### PR TITLE
[KYUUBI #3298] Unify the approach of get classLoader in Kyuubi

### PR DESCRIPTION
**Pull request by yikf**
### _Why are the changes needed?_

fix https://github.com/apache/incubator-kyuubi/issues/3298

Curently, a serval snippets use `getClass.getClassLoader` to get the classLoader in Apache Kyuubi, this pr aims to unify this behavior in `Utils.getContextOrKyuubiClassLoader`

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/benchmark/Benchmark.scala [hunk 1]
```diff
@@ -17,13 +17,6 @@
 
 package org.apache.kyuubi.tpcds.benchmark
 
-import scala.concurrent._
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.duration.DurationInt
-import scala.language.implicitConversions
-import scala.util.{Failure => SFailure, Success, Try}
-import scala.util.control.NonFatal
```
Comment (yikf) [inline dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/benchmark/Benchmark.scala:25]: `UnusedImports` are removed by spotless

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Hi @yikf, let's make the PR description contains more details, it will be included into commit message, and it's not necessary create an issue for it, a direct PR is fine as well.

**Comment by yikf** [comment]
> Hi @yikf, let's make the PR description contains more details, it will be included into commit message, and it's not necessary create an issue for it, a direct PR is fine as well.

got it, thanks

**Review by zhaomin1423 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master

File: dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/benchmark/Benchmark.scala (hunk 1 above)
Comment (pan3793) [reply dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/benchmark/Benchmark.scala:25]: @Yikf it's not unused imports, and it breaks the CI.

Comment (pan3793) [reply dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/benchmark/Benchmark.scala:25]: could you please send a follow-up PR to recover the build? otherwise, I need to revert this one.

Comment (pan3793) [reply dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/benchmark/Benchmark.scala:25]: FYI: you can use `./build/mvn clean package -Ptpcds -pl dev/kyuubi-tpcds -am` to verify if it works
